src/BloomBrowserUI/pageChooser/page-chooser.less

```diff
@@ -0,0 +1,161 @@
+// page-chooser.less - rules for the Bloom Page Chooser dialog
+// try not to overwrite the bloomDialog.less rules
+
+@PreviewPaneLeftPadding: 15px;
+BODY {
+	margin: 4px;
+}
+#mainContainer {
+	border: none;
+	margin: 0;
+	height: 645px;
+    max-width: 930px;
+	width: 100%;
+    display: flex;
+    flex-direction: row;
+	box-sizing: border-box;
+}
+.templatePageDisplay {
+	overflow-y: auto;
+	height: 681px;
```

Comment:
less will let us use symbols and arithmetic to indicate the meaning and relationships of these numbers, won't it?